File: jtemplate/benchmarks/Gcd.java
public class Gcd{

public static int gcd(int x,int y){
	if (x==y)
		return x;
	if (x<y)
		return gcd(x,y-x);
	else
		return gcd(x-y,x);
}

public static void main(String[] args){
	int a=0;
	for (int i=0;i<10000;++i)
		a=gcd(28388383,100101);
	System.out.println(a);
}

}

File: jtemplate/benchmarks/gcd.php
<?php
function gcd($x,$y){
	return $x<$y ? gcd($x,$y-$x) :( $x>$y ? gcd($x-$y,$x) : $x);
}

$a=0;
for ($i=0;$i<10000;$i+=1){
	$a=gcd(28388383,100101);
}
print_r ($a);
?>
